modules/kubernetes-addons/spark-k8s-operator/locals.tf

```diff
@@ -1,10 +1,15 @@
 
 locals {
   default_helm_config = {
-    name                       = "spark-operator"
-    chart                      = "spark-operator"
-    repository                 = "https://googlecloudplatform.github.io/spark-on-k8s-operator"
-    version                    = "1.1.6"
+    name       = "spark-operator"
+    chart      = "spark-operator"
+    repository = "https://googlecloudplatform.github.io/spark-on-k8s-operator"
+    version    = "1.1.15"
+    # image configuration can be removed once this issue is merged: https://github.com/GoogleCloudPlatform/spark-on-k8s-operator/pull/1454
+    image = {
+      repository = "ghcr.io/googlecloudplatform/spark-operator"
+      tag        = "v1beta2-1.3.3-3.1.1"
+    }
```

Comment:
I know this is a defect with latest helm chart for Spark Operator. I contributed to Spark Operator repo to fix that issue. This will be resolved in their next Helm Chart release. 

We don't want to add the workaround by passing the image details here. I would say keep the old version until we have a stable version from Spark Operator.

If users want to use the latest then they can do something like this

```hcl
  #---------------------------------------
  # ENABLE SPARK on K8S OPERATOR
  #---------------------------------------
  enable_spark_k8s_operator = true
  # Optional Map value
  spark_k8s_operator_helm_config = {
    name             = "spark-operator"
    chart            = "spark-operator"
    repository       = "https://googlecloudplatform.github.io/spark-on-k8s-operator"
    version          = "1.1.15"
    namespace        = "spark-k8s-operator"
    timeout          = "300"
    create_namespace = true
    values           = [templatefile("${path.module}/values.yaml", {})]
  }
```
`values.yaml`
```yaml
image:
  repository: ghcr.io/googlecloudplatform/spark-operator
  tag: "v1beta2-1.3.3-3.1.1"
```

Reply:
Google team has merged the PR now. You can just update the defaults to `1.1.18` now